src/Utils/Hooks.php

```diff
@@ -11,4 +11,6 @@ class Hooks
 	const AFTER_ORDER = 'ln_wc_after_order';
```

Comment:
I agree, I guess they work like Java interfaces? gonna research a little bit, thank you! ( I didn't know we had interfaces in php sorry! )